pkgs/by-name/ra/railway-wallet-bin/package.nix

```diff
@@ -0,0 +1,22 @@
+{ lib
+, appimageTools
+, fetchurl
+}:
+
+appimageTools.wrapType2 rec {
+  pname = "railway-wallet-bin";
+  version = "5.16.9";
+
+  src = fetchurl {
+    url = "https://github.com/Railway-Wallet/Railway-Wallet/releases/download/v${version}/Railway-linux-x86_64.AppImage";
+    hash = "sha256-UI/eJoQFtWzP/vMumJVp8BRylmUqRMTdbAJpFYDzsh8=";
+  };
+
+  meta = with lib; {
```

Comment:
`with lib;` is problematic antipattern. Use `lib.licenses` `with lib.maintainers;` below.

Reply:
Resolved in https://github.com/NixOS/nixpkgs/pull/320395/commits/0eccf2dee2ec9270e6be5a1551970fc8577ad585